models/person.rb

```diff
@@ -3,21 +3,20 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
+  # getters and setters
+  attr_reader :id, :rentals
```

Comment:
[OPTIONAL] You are getting `:rentals` with the getters method. But you also need a setters method for `:rentals`. Or you have a free chance of making both in one place by using `attr_accessor `